modules/2_nbde/nbde.tf

```diff
@@ -57,6 +57,23 @@ resource "openstack_compute_instance_v2" "tang" {
 # Extract the instance's IP addresses
 locals {
   tang_hosts = join(",", [for ts in openstack_compute_instance_v2.tang : ts.network[0].fixed_ip_v4])
+  tang = {
+    volume_count = lookup(var.tang, "data_volume_count", 0),
+    volume_size  = lookup(var.tang, "data_volume_size", 0)
```

Comment:
hum... this is to set a default? 
We should probably be 1
volume size should be our default?

Reply:
Added default values for both the count and size